samples/abn-sample/backend/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	MY_VERSION         = "MY_VERSION"
+	DEFAULT_VERSION = "v1"
+)
+
+func getVersion() string {
+	version, ok := os.LookupEnv(MY_VERSION)
+	if !ok {
+		version = DEFAULT_VERSION
+	}
+	return version
+}
+
+// implment /recommend endpoint returning value of VERSION env variable
+func recommend(w http.ResponseWriter, req *http.Request) {
```

Comment:
Returning a version here seems really confusing. This is mimicking a ml recommendation isn't it? Can it at least return some random number (or even a constant ... 17 for instance instead of the version)?
***
Version identifiers have a very specific role to play in the labels, and I putting version inside the backend code is misleading / confusing.

Reply:
Notice that this constant to be returned by /recommend can be set using the env variable (calling that a version is the confusing part).

Reply:
changed return value.